### DISCUSSION TITLE
fact question

### DISCUSSION THREAD

**Discussion by gpetrom** [pyinfra-dev/pyinfra#1048] [General]
Hi to all 

I want to use files.file fact can someone give me an example what should i import to work.
It will be useful for someone like me that it does not have so much experience with python to have a full example on every fact. 

thanks in advanced

**Comment by maisim** [comment]
Hi !

You can have a look in the examples directory :
https://github.com/pyinfra-dev/pyinfra/tree/2.x/examples

a Fact file usage example:
https://github.com/pyinfra-dev/pyinfra/blob/2.x/examples/files.py
